PE/__init__.py

```diff
@@ -454,7 +454,13 @@ def getDllName(self):
         '''
         if self.IMAGE_EXPORT_DIRECTORY is not None:
             rawname = self.readAtRva(self.IMAGE_EXPORT_DIRECTORY.Name, 32)
-            return rawname.split(b'\x00')[0].decode('utf-8')
+            if not rawname:
+                return None
+
+            try:
+                return rawname.partition(b'\x00')[0].decode('ascii')
```

Comment:
https://docs.microsoft.com/en-us/windows/win32/debug/pe-format#export-directory-table

![image](https://user-images.githubusercontent.com/156560/128802710-a1f638f4-4ee6-4e36-b5d4-4e44cb6d981d.png)

Reply:
`partition` because it only splits once and therefore better indicates the meaning